src/atomate2/lobster/schemas.py

```diff
@@ -304,6 +308,90 @@ def from_directory(
             return (None, None, None, None, None)
 
 
+class CalcQualitySummary(BaseModel):
+    """Model describing the calculation quality of lobster run."""
+
+    minimal_basis: bool = Field(
+        None,
+        description="Denotes whether the calculation used the minimal basis for the "
+        "LOBSTER computation",
+    )
+    charge_spilling: dict = Field(
+        None,
+        description="Dict contains the absolute charge spilling value",
+    )
+    charges: dict = Field(
+        None,
+        description="Dict contains the LOBSTER and BVA charge sign comparison results",
+    )
+    band_overlaps: dict = Field(
+        None,
+        description="Dict summarizing important information from the "
+        "bandOverlaps.lobster file to evaluate the quality of the projection, "
+        "namely whether the file is generated during projection (i.e., larger "
+        "deviations exist), the maximum deviation observed, percent of k-points "
+        "above the threshold set in the pymatgen parser (during data generation the "
+        "value was set to 0.1)",
+    )
+    dos_comparisons: dict = Field(
```

Comment:
Do the four `dict` fields in `CalcQualitySummary` have an expected schema or are they free-form? If the former, we should probably enforce it in the model.

Reply:
Thanks! They have an expected form, I can create a model for that as well 😃

Reply:
@tschaume , have now added models for the same as per your suggestion 😄

Reply:
I will add that any other remaining fields that are validated as a raw `dict` should also have an explicit model if the intention is to query the data down the line.

Reply:
Done